cockroachcloud/managing-cmek.md

```diff
@@ -96,7 +157,86 @@ curl --request GET \
   --header "Authorization: Bearer ${API_KEY}"
 ```
 
-## Revoking CMEK for a cluster
+## Rotate a CMEK key
+
+{% include cockroachcloud/cmek-rotation-types.md %}
+
+The API to rotate a CMEK key is nearly identical to the API to [activate CMEK on a cluster](#step-4-activate-cmek-for-your-cockroachdb-dedicated-cluster), with one notable exception. When you activate CMEK, you use a `POST` request that includes a CMEK key for each of the cluster's regions. When you rotate a CMEK key, you use a `PUT` request that includes a CMEK key for each region you intend to rotate.
```

Comment:
```suggestion
The API to rotate a CMEK key is nearly identical to the API to [activate CMEK on a cluster](#step-4-activate-cmek-for-your-cockroachdb-dedicated-cluster), with one notable exception. When you activate CMEK, you use a `POST` request that includes a CMEK key for each of the cluster's regions. When you rotate a CMEK key, you use a `PUT` request that includes a CMEK key for each region where you intend to rotate.

Note: Do not disable or delete the previous CMEK key until the rotation process is complete.
```

Reply:
Should we mention that `POST` can also be used to rotate, as long as it has all regions? Or would that confuse the narrative?

Reply:
I would prefer to skip that. It's already a lot of nuance to absorb :)